csharp/src/Drivers/Apache/Spark/SparkConnection.cs

```diff
@@ -134,23 +134,6 @@ public override AdbcStatement CreateStatement()
             return new SparkStatement(this);
         }
 
-        public override void Dispose()
-        {
-            /*
-            if (this.client != null)
-            {
-                TCloseSessionReq r6 = new TCloseSessionReq(this.sessionHandle);
-                this.client.CloseSession(r6).Wait();
-
-                this.transport.Close();
-                this.client.Dispose();
-
-                this.transport = null;
-                this.client = null;
-            }
-            */
-        }
-
         public override IArrowArrayStream GetInfo(IReadOnlyList<AdbcInfoCode> codes)
```

Comment:
We are going to need to close the session on the Spark side or the resources will remain in use. Please file a followup bug for this.

Reply:
Just to be clear, if you're referring to the removed `Dispose`, there is a correct implementation of the `Dispose` in the base class `HiveServer2Statement`.